src/README.MD

```diff
@@ -0,0 +1,9 @@
+Using the CoreWCF Postman Samples Collection.json within Postman
```

Comment:
Can you move this file to the src/Samples folder instead of being at the top level